thefederation/views.py

```diff
@@ -9,12 +9,14 @@
 
 
 def register_view(request, host):
+    json = True if request.content_type == "application/json" else False
```

Comment:
You can also write this a bit more pythonic as `json = request.content_type == "application/json`. But no need to change, just a note :)